cmd/cone/config.go

```diff
@@ -15,8 +15,17 @@ const (
 	envPrefix = "cone"
 )
 
+func defaultConfigPath() string {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		homeDir = "$HOME"
```

Comment:
I think that if the homedir can't be looked up, it is safer to use the working directory `.`. `os.UserHomeDir()` errors if something like `$HOME` isn't in the environment, which seems more than possible in a docker type situation.

Reply:
Also, I'm not sure there is anything to expand `$HOME` from here, so I think it may just result in an invalid path.